src/windows.jl

```diff
@@ -159,7 +160,7 @@ The window is defined by sampling the continuous function:
 
 in the range `[-0.5, 0.5]`
 
-The `hanning` window satisfies the Constant Overlap-Add (COLA) property with an
+The `hanning` window satisfies the Constant Overlap-Add (COLA) property with a
```

Comment:
Perhaps explain in the documentation why it is exported as hann too (similar to the explanation in the PR). 1) it's interesting piece of knowledge, and 2) so users can find it.